static/js/main.js

```diff
@@ -36,6 +36,21 @@ const getToolbox = () => {
 	}
 };
 
+const options = {
```

Comment:
nit: rename this to `workspaceOptions` or something.

Reply:
Followed